### DISCUSSION TITLE
Can not write ConfigMaps of services created from an Application

### DISCUSSION THREAD

**Discussion by alexBonadei** [mia-platform/community#193] [Q&A] [answered]
[](url)Hi,

I've just installed a new application in the console, which creates a couple of microservices.
I need to read and eventually write the ConfigMaps of such microservices, but in the ConfigMap section this message is displayed.

![Screenshot from 2023-02-07 14-37-47](https://user-images.githubusercontent.com/67758444/217260107-e41156ff-57fc-4f7e-aede-bc91fdd16b31.png)

Even if I saved many times, I still can't see the ConfigMap.

Am I missing something?
Thank you in advance

**Comment by Danielecina** [comment, marked as answer]
Hi @alexBonadei, the problem is caused by the erroneous configuration of the application that has set the data-visualization-backend config map as ReadOnly. 

In the meantime you can fix this problem quickly by following this step:
- Go to your git provider by clicking on top of your project
- Enter in the api-console-config file
- Find the microservice data-visualization-backend
- Change the configMap data-visualization field viewAsReadOnly to false

**Reply by Danielecina** [reply]
In the meantime, we will fix the problem as soon as possible.

**Reply by alexBonadei** [reply]
Fixed! Thank you very much!
